### PR TITLE
delete functionality without authentication

### PR DESCRIPTION
**Pull request by sharmashalu**
Delete user without authentication.

### CODE REVIEW

File: expressEngine/user/service/user.js [hunk 1]
```diff
@@ -31,6 +31,19 @@ class Service extends BaseService {
       return this.error(err);
     }
   }
+  deleteUser = async (id) => {
+    try {
+      const isExist = await DBService.findOne(User, {_id: id});
+      if(!isExist) {
+        console.log('1st');
```
Comment (manish-nx) [inline expressEngine/user/service/user.js]: unnecessary console statement

File: expressEngine/user/service/user.js [hunk 2]
```diff
@@ -31,6 +31,19 @@ class Service extends BaseService {
       return this.error(err);
     }
   }
+  deleteUser = async (id) => {
+    try {
+      const isExist = await DBService.findOne(User, {_id: id});
```
Comment (manish-nx) [inline expressEngine/user/service/user.js]: Create a variable for {_id: id}, so that we need to change it only once if required in the future.

Comment (himanshu-nodex) [reply expressEngine/user/service/user.js]: Unnecessarily using two queries for deletion can be done using only one:  findOneAndDelete().
Please do so.

**Comment by himanshu-nodex** [comment]
Fixed the comments on feature/18

**Pull request closed without merging**